apps/generator/docs/deprecate-cli.js-ag.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Deprecate ag"
+weight: 170
+---
+
+# Migration Guide from “ag/asyncapi-generator” to AsyncAPI CLI
```

Comment:
no need for h1 here, this should be handled by `title`, so please remove it and make nicer title